### PR TITLE
Fix TypeScript errors in the Playground app

### PR DESCRIPTION
**Pull request by guyca**
The TS migration has surfaced a lot of errors and missing options. This PR aims to address the wrong and missing option declarations and all compiler issues in the playground app.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,7 +67,10 @@
     "@babel/plugin-proposal-export-default-from": "7.10.1",
     "@babel/plugin-proposal-export-namespace-from": "7.10.1",
     "@babel/types": "7.6.x",
+    "@react-native-community/blur": "^3.6.0",
```
Comment (ItsNoHax) [inline package.json:70]: I don't see this being used?

File: package.json [hunk 2]
```diff
@@ -67,7 +67,10 @@
...
+    "@react-native-community/datetimepicker": "^2.5.0",
```
Comment (ItsNoHax) [inline package.json:71]: Not being used?

File: package.json [hunk 3]
```diff
@@ -67,7 +67,10 @@
...
     "@react-native-community/eslint-config": "2.0.0",
+    "@react-native-community/netinfo": "^5.9.4",
```
Comment (ItsNoHax) [inline package.json:73]: Not being used

File: package.json [hunk 4]
```diff
@@ -94,8 +97,10 @@
     "prettier": "2.0.5",
     "react": "16.11.0",
     "react-native": "0.62.2",
+    "react-native-gesture-handler": "^1.6.1",
```
Comment (ItsNoHax) [inline package.json:100]: not being used

File: package.json [hunk 5]
```diff
@@ -94,8 +97,10 @@
...
     "react-native-keyboard-tracking-view": "5.x.x",
-    "react-native-ui-lib": "3.24.2",
+    "react-native-reanimated": "^1.9.0",
```
Comment (ItsNoHax) [inline package.json:102]: not being used

File: package.json (hunk 1 above)
Comment (guyca) [reply package.json:70]: It's not - but it's a peer dependency of react-native-ui-lib which we had to upgrade. I hope we can get rid of this soon in their v6

**Pull request merged**